.github/workflows/serverless-integration.yml

```diff
@@ -51,7 +51,7 @@ jobs:
           AWS_ACCESS_KEY_ID: ${{ secrets.SERVERLESS_AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.SERVERLESS_AWS_SECRET_ACCESS_KEY }}
         with:
-          timeout_minutes: 30
+          timeout_minutes: 45
```

Comment:
Sometimes the first run of the integration test will timeout (it needs about 8 more minutes). Bumping to 45 fixes this